resources/utility-scripts/wait_for_startup.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+DB_NUMBER=9
+
+#check if the number of connections to the postgresql server is at least as much as DB_NUMBER to detect that the startup phase has begun
+while true; do
+  echo "Starting MedCo up..."
+  DB_CONNECTIONS_NUMBER=$(PGPASSWORD=postgres psql -v ON_ERROR_STOP=1 -h localhost -p 5432 -U postgres -d postgres -t -c "SELECT count(*) FROM pg_stat_activity WHERE usename!='postgres';")
```

Comment:
The database connection infos should also be configurable through .travis.yml file (again, if possible). This will also prepare the ground to be able to test the test-local-3nodes profile later.